frontend/src/views/SettingSidebar.vue

```diff
@@ -83,6 +83,11 @@
             class="outline-item group w-full flex items-center pl-11 pr-2 py-2"
             >{{ $t("settings.sidebar.subscription") }}</router-link
           >
+          <router-link
```

Comment:
Let's make this visible to OWNER and DBA for now

Reply:
We also need to add a check at router/index.ts https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/frontend/src/router/index.ts?L989&subtree=true

Otherwise, developer can still access it with the url.

Reply:
Also added limit on the server side. [`7aa1157` (#2473)](https://github.com/bytebase/bytebase/pull/2473/commits/7aa11578693eb76b09a5e42e20c7fd1439dddc56)